cypress/fixtures/messages/downloadableImage.json

```diff
@@ -0,0 +1,39 @@
+{
+    "text": null,
+    "data": {
+        "_cognigy": {
+            "_default": {
+                "_image": {
+                    "type": "image",
+                    "imageUrl": "https://placekitten.com/300/300",
+                    "buttons": [
+                        {
+                            "id": 0.17194659386513078,
+                            "payload": "",
```

Comment:
the id and payload fields are not used or are they? I didn't see them in your other PR. are they needed for the test?

Reply:
Yes, they were added for tests. I first wrote a test that uses the id. But I will remove this as it is not used anymore

Reply:
removed